docs/ui/components/VideoBoxLink/index.tsx

```diff
@@ -24,30 +24,40 @@ export function VideoBoxLink({ title, description, videoId, className }: VideoBo
         '[&+hr]:!mt-6',
         className
       )}
-      isStyled>
+      isStyled
+      aria-label={`Watch video: ${title} (opens in new tab)`}
+      role="link">
```

Comment:
```suggestion
      aria-label={`Watch video: ${title} (opens in new tab)`}>
```

Role `link` on link should be set by default by browser engine.